integrations/haproxy/documentation.yaml

```diff
@@ -83,3 +83,5 @@ podmonitoring_config: |
       matchLabels:
         app.kubernetes.io/name: haproxy
 sample_promql_query: up{job="haproxy", cluster="{{cluster_name}}", namespace="{{namespace_name}}"}
+alerts_config: {% includecode content_path="stackdriver/docs/managed-prometheus/exporters/configs/haproxy_alerts.yaml" %}
```

Comment:
I'm worried about having a file called _alerts.yaml outside of the top-level alerts directory of this repo (meant to contain templates for Cloud alerting, not prometheus). Would it be possible to embed the file content here as a string?

Reading around online it should be possible using the `|` yaml string delimiter and some additional indentation. i.e. https://medium.com/@george.shuklin/embedding-yaml-into-yaml-3511246d7ea7

Reply:
Not a problem, done.

Reply:
Great, thank you!